src/tests/fixtures/trip.fixture.js

```diff
@@ -82,4 +94,79 @@ export const createTrip = async () => {
   return dataValues;
 };
 
-export default newTrip;
+export const createTrips = async () => {
+  const testTrip = {
+    tripType: faker.random.arrayElement(['one-way']),
+    tripId: pendingRequest.tripId,
+    userId: pendingRequest.requesterId,
+    originId: faker.random.arrayElement([1]),
+    destinationId: faker.random.arrayElement([2]),
+    departureDate: faker.random.arrayElement(['2020-05-20', '2020-05-22']),
+    travelReasons: faker.lorem.sentence(),
+    accommodationId: faker.random.number()
+  };
+  const acceptedRequestTrip = {
+    tripId: acceptedRequest.tripId,
+    userId: acceptedRequest.requesterId,
+    originId: faker.random.arrayElement([1]),
+    destinationId: faker.random.arrayElement([2]),
+    departureDate: faker.random.arrayElement(['2020-05-20', '2020-05-22']),
+    travelReasons: faker.lorem.sentence(),
+    accommodationId: faker.random.number()
+  };
+
+  const rejectedRequestTrip = {
+    tripId: rejectedRequest.tripId,
+    userId: rejectedRequest.requesterId,
+    originId: faker.random.arrayElement([1]),
+    destinationId: faker.random.arrayElement([2]),
+    departureDate: faker.random.arrayElement(['2020-05-20', '2020-05-22']),
+    travelReasons: faker.lorem.sentence(),
+    accommodationId: faker.random.number()
+  };
+
+  await Trip.destroy({ where: {} });
+  const { dataValues } = await TripService.createTrip(testTrip);
+  await TripService.createTrip(acceptedRequestTrip);
+  await TripService.createTrip(rejectedRequestTrip);
+  return dataValues;
+};
+
+export const newCorrectlyEditedTrip = {
+  originId: faker.random.arrayElement([2]),
+  destinationId: faker.random.arrayElement([3]),
+  departureDate: faker.random.arrayElement(['2020-10-20', '2020-10-26']),
+  travelReasons: faker.lorem.sentence(),
+  accommodationId: faker.random.number()
+};
+
+export const newInCorrectlyEditedTrip = {
+  originId: faker.random.arrayElement([2]),
+  destinationId: faker.random.arrayElement([3]),
+  departureDate: faker.random.arrayElement(['2020-10-20', '2020-10-26']),
+  travelReasons: '',
+  accommodationId: faker.random.number()
+};
+
+export const newInCorrectLocationTrip = {
+  originId: faker.random.arrayElement([10]),
+  destinationId: faker.random.arrayElement([3]),
+  departureDate: faker.random.arrayElement(['2020-10-20', '2020-10-26']),
+  travelReasons: faker.lorem.sentence(),
+  accommodationId: faker.random.number()
+};
+
+export const newSameLocationsTrip = {
+  originId: faker.random.arrayElement([1]),
+  destinationId: faker.random.arrayElement([1]),
+  departureDate: faker.random.arrayElement(['2020-10-20', '2020-10-26']),
+  travelReasons: faker.lorem.sentence(),
+  accommodationId: faker.random.number()
+};
```

Comment:
There is no much difference between these objects and `newCorrectlyEditedTrip`. Use spread operator instead and override the property you want to change.